pkgs/games/MAR1D/default.nix

```diff
@@ -0,0 +1,53 @@
+{stdenv, fetchFromGitHub, cmake, mesa_glu, x11, xorg, xinput_calibrator, doxygen, libpthreadstubs, alsaLib, alsaOss, libao}:
+
+stdenv.mkDerivation rec {
+  name = "MAR1D-${version}";
+  version = "0.1.0";
+  width = 30;
+  mute = false;
+  effects = false;
+  sensitivity = 5;
+  reverseY = false;
+  options = "-w${toString width}"
+          + " -s${toString sensitivity}"
+          + (if mute then " -m" else "")
+          + (if effects then " -f" else "")
+          + (if reverseY then " -r" else "");
+
+  src = fetchFromGitHub {
+    sha256 = "0x5a9kgzggdybphbgc7969kbrynxgdhk6481irgssizihkjj41bj";
+    rev = "v${version}";
+    repo = "fp_mario";
+    owner = "olynch";
+  };
+
+  buildInputs = [ cmake mesa_glu x11 xorg.libXrandr xorg.xinput xinput_calibrator doxygen libpthreadstubs xorg.libXi alsaLib alsaOss libao ];
+
+  configurePhase = ''
+    mkdir build
+    cd build
+    cmake ../src
+  '';
+
+  buildPhase = ''
+    make
+    cd ..
+  '';
```

Comment:
Are the configure and build phases required? I think the cmake setup hook should work fine for this case? If not I'd ping the nixpkgs cmake maintainer and see why it isn't working for you.